demisto_sdk/commands/lint/templates/dockerfile.jinja2

```diff
@@ -25,3 +27,12 @@ RUN pwsh -Command Install-Module -Name Pester -Scope AllUsers -Force | pwsh -Com
 RUN pwsh -Command Install-Module -Name PSScriptAnalyzer -Scope AllUsers -Force | pwsh -Command Out-Null
 {# Container entry point (Every command will start with /bin/sh) #}
 {% endif %}
+{% else %}
+{# Creating test main directory #}
+RUN mkdir -p /devwork/
+{# Define workdir #}
+WORKDIR /devwork
```

Comment:
Duplicate step, done also in first steps, both for pwsh and python

Reply:
We already did it when we created the image in the first place, in the first build section above, it already created, so its redundant step.
https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/lint/linter.py#L528